.github/workflows/all-checks.yml

```diff
@@ -0,0 +1,18 @@
+name: All checks pass
+on:
+    pull_request:
+    push:
+      branches:
+        - main
+jobs:
+    allchecks:
+        runs-on: ubuntu-latest
+        permissions:
+            checks: read
+            contents: read
+        steps:
+          - uses: wechuli/allcheckspassed@v1
+            with:
+                delay: '3'
+                retries: '30'
```

Comment:
30 retries?

Reply:
It polls once a minute to see if the PR has succeeded, up to `retries` times.  Our seems to take a bit less than 20 minutes: 30 seemed like a reasonable buffer